Makefile

```diff
@@ -10,27 +11,39 @@ datarootdir = $(prefix)/share
 mandir = $(prefix)/share/man
 man1dir = $(mandir)/man1
 
-SOURCES = minecraftd.sh.in minecraftd.conf.in minecraftd.service.in minecraftd.sysusers.in minecraftd.tmpfiles.in minecraftd-backup.service.in minecraftd-backup.timer.in
+SOURCES = $(wildcard *.in)
 OBJECTS = $(SOURCES:.in=)
 
 GAME = minecraft
 INAME = minecraftd
 SERVER_ROOT = /srv/$(GAME)
+CONFIG_PATH = /etc/conf.d/$(GAME)
+SYSCONFDIR = /etc
+INSTANCE_CONFIG_DIR = $(SYSCONFDIR)/$(GAME)
 BACKUP_DEST = $(SERVER_ROOT)/backup
+LIBRARY_PATH = $(libdir)/$(GAME)
 BACKUP_PATHS = world
 BACKUP_FLAGS = -z
 KEEP_BACKUPS = 10
 GAME_USER = $(GAME)
+SERVER_MEMORY_INITIAL = 512
+SERVER_MEMORY_MAXIMUM = 1024
 MAIN_EXECUTABLE = minecraft_server.jar
+ifeq ($(MAIN_EXECUTABLE),$(MAIN_EXECUTABLE:/%=))
+MAIN_EXECUTABLE_ABSOLUTE = $(SERVER_ROOT)/$(MAIN_EXECUTABLE)
+else
+MAIN_EXECUTABLE_ABSOLUTE = $(MAIN_EXECUTABLE)
+endif
+TMUX_SOCKET_DIR = /run/$(GAME)/tmux
 SESSION_NAME = $(GAME)
-SERVER_START_CMD = java -Xms512M -Xmx1024M -jar ./$${MAIN_EXECUTABLE} nogui
+SERVER_START_CMD = java -Xms@SERVER_MEMORY_INITIAL@M -Xmx@SERVER_MEMORY_MAXIMUM@M -jar @MAIN_EXECUTABLE@ nogui
```

Comment:
Yeah, while it does work, I'm open to any better solution.

My goal here is to allow users to customize the memory allocated to the server without overriding the entire JVM command line, should the packager need to add/change/remove other JVM arguments. Do you have any recommendations for how this could be accomplished? I'm okay with returning to the use of `eval` if you think that's better.

Reply:
The goal is clear but I don't like adding any java specific options. The only viable solution to me seems to alter all of `SERVER_START_CMD`.

I am afraid you have to jog my memory a little about your proposal regarding `eval`. In which way was `eval` used before?